src/lib/data/blogs/getting-started-with-blend.ts

```diff
@@ -0,0 +1,23 @@
+import blendLogo from '$lib/assets/blend_logo.png';
+const post: Blog.BlogPost = {
```

Comment:
Blogs are written by combining these blocks and providing some metadata

Reply:
nice love it